requirements_all.txt

```diff
@@ -78,7 +78,7 @@ TravisPy==0.3.5
 TwitterAPI==2.5.9
 
 # homeassistant.components.sensor.waze_travel_time
-WazeRouteCalculator==0.9
+WazeRouteCalculator==0.6
```

Comment:
I don't know *what* happened here, but please re-run the gen_requirements_all script. This file contains changes from other prs (or reverts)